weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -182,7 +182,7 @@ const useMakeTraceServerEndpoint = <
   return traceServerRequest;
 };
 
-const useCall = (params: UseCallParams): Loadable<CallSchema | null> => {
+export const useCall = (params: UseCallParams): Loadable<CallSchema | null> => {
```

Comment:
Nice - I actually like that you started exporting these. Adding this for context sharing: normally, everywhere you would do something like `const {useCall} = useWFHooks();`, then call `useCall`. This is because when we first started Weave, we had an adapter layer for our two backends (yes, we had a FastAPI backend AND a QueryService backend). The query service one can/should be deprecated which means the adapter can go away, which means we can just use these methods directly.